spec/coordinates_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "../lib/coordinates"
+
+RSpec.describe Coordinates do
+  describe "#valid?" do
+    it "is true when the value is in the grid" do
+      expect(Coordinates.new(value: "A1")).to be_valid
+      expect(Coordinates.new(value: "B2")).to be_valid
+      expect(Coordinates.new(value: "C3")).to be_valid
+    end
+
+    it "is false when the value is not in the grid" do
+      expect(Coordinates.new(value: "D1")).not_to be_valid
```

Comment:
I would call out what about each one of these examples is not acceptable. So we don't have to know about the game coordinates to understand at a glance. So I would recommend that each example be its own spec, with a good description of why it is invalid.

Reply:
Oh, that's a good idea.

Reply:
Updated in f97bc00fc59611a7fc8cd418987d6aec5c6e738e